api/client/out.go

```diff
@@ -0,0 +1,69 @@
+package client
+
+import (
+	"io"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/term"
+)
+
+// OutStream is an output stream used by the DockerCli to write normal program
+// output.
+type OutStream struct {
```

Comment:
We even could have a `Stream` struct that have comment field and method from `InStream` and `OutStream`, wdyt ?